### ISSUE TITLE
Snapshot unavailable though listed in CDX and /wayback/available

### ISSUE THREAD

**Issue by Kevin-Prichard** [internetarchive/wayback#210]
A snapshot timestamp returned by both /wayback/available and CDX is unavailable when requested at web.archive.org/web/$timesetamp/$url

e.g. queries on both index services return the same timestamp-

Wayback: https://archive.org/wayback/available?url=latimes.com

Body: {"archived_snapshots": {"closest": {"status": "200", "available": true, "url": "http://web.archive.org/web/20200119032503/https://www.latimes.com/", "timestamp": "20200119032503"}}, "url": "latimes.com"}

CDX: http://web.archive.org/cdx/search/cdx?url=latimes.com&output=json&from=20200119000000&to=20200119235959

Body: ['com,latimes)/', '20200119032503', 'https://www.latimes.com/', 'text/html', '200', '37KBTKK4TTHIZ3WUQTZJVQQSBHISBSOC', '55683']

However, when requesting the indicated URL-
http://web.archive.org/web/20200119032503/https://www.latimes.com/
-in a browser (today, at 20200119@1630 UTC) we get redirected to an *earlier* snapshot, 20200115194259.

Is this expected behavior?  Why isn't the snapshot indicated in query results available in wayback?

Is there some latency in the machinery that we need to be aware of?

**Comment by akamhy** [comment]
I did not wrote a single line of wayback but I've helped folks archive millions of web pages on Wayback Machine and created [waybackpy](https://github.com/akamhy/waybackpy) so I've some experience and what you were experiencing was most likely a database lag. The URL (http://web.archive.org/web/20200119032503/https://www.latimes.com/) doesn't redirect anymore.

Is this expected behavior?
- Yes, I've experienced this more than 10K times. Just trying to grab the webpage source right after a recent capture is usually sufficient to reproduce this issue.

Why isn't the snapshot indicated in query results available in wayback?
- Replication lag

Is there some latency in the machinery that we need to be aware of?
- Yes, Replication lag.
